src/main/java/BaseBallGame.java

```diff
@@ -0,0 +1,107 @@
+public class BaseBallGame {
+    final int chance;
+    final String randomNum;
+
+    public BaseBallGame(int chance, String randomNum) {
+        this.chance = chance;
+        this.randomNum = randomNum;
+    }
+
+    public void run() {
+        int attempt = 1;
+        while (attempt <= chance) {
+            String response = new CustomScanner().getResponse(chance + "기회 중 " + attempt + "번째 시도입니다. 던지세요.");
+            if (!isNumber(response)) {
+                continue;
+            }
+            if (!sameLength(response, randomNum)) {
+                continue;
+            }
+            if (confirm(response)) {
+                success();
+                break;
+            }
+            if (attempt == chance) {
+                fail();
+            }
+            attempt++;
+        }
+    }
+
+    private boolean isNumber(String response) {
+        boolean isNumber = true;
+        try {
+            Integer.parseInt(response);
+        } catch (NumberFormatException e) {
+            System.out.println("잘못된 입력입니다. 숫자만 입력하여 주세요.");
+            isNumber = false;
+        }
+        return isNumber;
+    }
+
+    private boolean sameLength(String response, String randomNum) {
```

Comment:
```suggestion
    private boolean isSameLength(String response, String randomNum) {
```
<br>
메서드 이름은 동사로 시작해야합니다.